config/s2i/influxdb/continuous-infra-create-openshift-project

```diff
@@ -0,0 +1,21 @@
+PROJECT=continuous-infra
```

Comment:
I don't think this file will work in its proposed place because there are no jenkins templates at its level and it looks like a dupe of the one in config/s2i.. Is this file meant to be here?

Reply:
Good catch. This shouldn't be there.